### PR TITLE
Add party to tests

### PR DESCRIPTION
**Pull request by nbdavies**
The diff is looking messy, but here's the gist:
* Party for Scattering was left blank, except for partisan primaries (in which we have Democratic Scattering, Republican Scattering, etc.).
* For non-partisan races, people candidates (i.e. not Scattering) got party "NP", except as noted in https://github.com/openelections/openelections-data-wi/issues/49.
* For partisan races, people candidates' parties were looked up in Wisconsin Elections Commission summary PDFs for the corresponding election.
* In a couple instances, we had lines with party and lines without party split out into separate blocks with the same file name. `20001107__wi__general__ward` is an example of this. Those no longer need to be separate blocks for the sake of feature test formatting, so that duplicate file heading was removed.

### CODE REVIEW

File: tests/features/wi-elections.feature.csv [hunk 1]
```diff
@@ -146,7 +146,7 @@ filename,party,candidate,county,office,district,ward,votes,total
 20110809__wi__general-recall__ward.csv,,,,,,,,
 ,REP,ROBERT L. COWLES,Brown,State Senate,2,CITY OF DE PERE District 1 Ward 1-3,1057,2016
 ,DEM,NANCY J. NUSBAUM,Outagamie,State Senate,2,VILLAGE OF SHIOCTON Village of Shiocton,86,180
-,REP,SANDRA K. PASCH,Washington,State Senate,8,CITY OF MILWAUKEE Ward 262,0,0
+,DEM,SANDRA K. PASCH,Washington,State Senate,8,CITY OF MILWAUKEE Ward 262,0,0
```
Comment (davipo) [inline tests/features/wi-elections.feature.csv:149]: Why is party changed here? How does this test pass, or why did it pass before the change?
(Oh, I see, the party was added incorrectly in the previous commit.)

**Review by davipo — APPROVED** [review]

File: tests/features/wi-elections.feature.csv (hunk 1 above)
Comment (nbdavies) [reply tests/features/wi-elections.feature.csv:149]: Yeah, just human data entry error

> why did it pass before the change?

I thought I saw that, but I wasn't sure how to get back to it. Might be worth checking that Travis correctly recognizes a test failure as a build failure

Comment (davipo) [reply tests/features/wi-elections.feature.csv:149]: Good point, thanks!
run_spot_tests.py must exit with a non-zero return code to fail the build.
Fixed that, and tested that errors in tests fail the build.
(Introduced 8 errors: checking adding, changing, or removing party from Scattering records, changing capitalization of candidate and county  -- all were caught.)
